.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,18 @@
+name: CI
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
Specify the current Ubuntu LTS instead of "latest" to avoid unexpected behavior.